lms/urls.py

```diff
@@ -87,7 +152,7 @@
     url(r'^api/val/v0/', include('edxval.urls')),
 
     url(r'^api/commerce/', include('commerce.api.urls', namespace='commerce_api')),
-    url(r'^api/credit/', include('openedx.core.djangoapps.credit.urls', app_name="credit", namespace='credit')),
+    url(r'^api/credit/', include('openedx.core.djangoapps.credit.urls', namespace='credit')),
```

Comment:
Why remove app_name?  It seems to be a no-op since the instance namespace is set to the same value, so I can understand the desire to cleanup, but app_name isn't also removed in other URLconfs such as support.urls and certificates.urls.

Reply:
Hrm, the other app_names should have been deleted too since they're replaced in 1.9 with app_name inside the included urls file. Just pushed that.